package.json

```diff
@@ -141,5 +149,45 @@
       ]
     }
   },
+  "oclif": {
+    "bin": "rdme",
+    "dirname": "rdme",
+    "commands": {
+      "strategy": "explicit",
+      "target": "./dist/index.js",
+      "identifier": "COMMANDS"
+    },
```

Comment:
Why not let oclif automatically pick up commands?

Reply:
we bundle the `src/index.ts` (i.e., `dist/index.js`) entrypoint for our github action bundles — see the `rollup.config.js` changes in #1067